### ISSUE TITLE
MetadataSet proto causing failure to load on previously (recently)  working world

### ISSUE THREAD

**Issue by splace** [create3000/x_ite#138]
```
x_ite.min.js:1 Error: Unknown node type or proto 'MetadataSet', you probably have insufficient component/profile statements, and/or an inappropriate specification version.
    at hs.node (x_ite.min.js:1:392145)
    at hs.nodeStatement (x_ite.min.js:1:384466)
    at hs.sfnodeValue (x_ite.min.js:1:403939)
    at hs.fieldValue (x_ite.min.js:1:396409)
    at hs.nodeBodyElement (x_ite.min.js:1:395518)
    at hs.nodeBody (x_ite.min.js:1:394487)
    at hs.node (x_ite.min.js:1:392613)
    at hs.nodeStatement (x_ite.min.js:1:384730)
    at hs.statement (x_ite.min.js:1:384323)
    at hs.statements (x_ite.min.js:1:384135)
(anonymous) @ x_ite.min.js:1
x_ite.min.js:1 Couldn't load URL 'https://splace.github.io/3d/2005/mountain/world/mountain world.wrl': 
********************************************************************************
Parser error at line 47:35
in 'https://splace.github.io/3d/2005/mountain/world/mountain%20world.wrl'

WorldInfo {
  metadata DEF Titania MetadataSet {
                                  ^
Unknown node type or proto 'MetadataSet', you probably have insufficient component/profile statements, and/or an inappropriate specification version.
********************************************************************************

error @ x_ite.min.js:1
x_ite.min.js:1 Uncaught (in promise) Error: Couldn't load X3D file.
    at o.createX3DFromURL.o.ready (x_ite.min.js:1:1357910)
    at vl.createX3DFromURLAsync (x_ite.min.js:1:659644)
    at vl.loadDocumentError (x_ite.min.js:1:661255)
    at x_ite.min.js:1:380746
```

**Comment by create3000** [comment]
You should consider using X3D .x3dv format, because VRML does only support a limited set of X3D nodes.

**Comment by splace** [comment]
this world must have been passed though Titania, at some point, i wasn't aware/wanted the additions, it had been just working. i will revert to the original file.

this kind of stuff was why i had previously used a fixed version of x-ite, but that broke #120 for some worlds for me, so i switched to latest, which presumable resulted in this. should i fix on a major versions? or something?

**Comment by create3000** [comment]
Even better than `.x3dv` is `.x3d` format, because the file can be loaded even if there are some errors like 'node not found' or errors with routes, and others. This is because `.x3d` is an XML format. The file is then loaded as good as possible, which can in some cases fine, for instance if a Metadata* node is not found.
